shell/src/main/java/org/crsh/text/renderers/LogRecordRenderer.java

```diff
@@ -58,11 +58,11 @@ public LineRenderer next() {
             LogRecord record = stream.next();
             String line = formatter.format(record);
             Color color;
-            if (record.getLevel() == Level.SEVERE) {
+            if (Level.SEVERE.equals(record.getLevel())) {
```

Comment:
All of these in this file are for enums.  It's better to keep the simpler form of equality checking.